utils/config.ts

```diff
@@ -19,14 +22,20 @@ export const IConfig = t.interface({
 
   JWT_SUPPORT_TOKEN_PUBLIC_RSA_CERTIFICATE: NonEmptyString,
 
+  ADB2C_ADMIN_GROUP_NAME: NonEmptyString,
+
   ADB2C_CLIENT_ID: NonEmptyString,
+
+  ADB2C_CLIENT_KEY: NonEmptyString,
   ADB2C_POLICY_NAME: NonEmptyString,
+  ADB2C_TENANT_ID: NonEmptyString,
+
   ADB2C_TENANT_NAME: NonEmptyString,
 
+  ADB2C_CONFIG: t.any, // TODO: Define the ADB2C_CONFIG type
   DASHBOARD_LOGS_TABLE_NAME: NonEmptyString,
   DASHBOARD_STORAGE_CONNECTION_STRING: NonEmptyString,
-
-  ADB2C_CONFIG: t.any // TODO: Define the ADB2C_CONFIG type
+  IN_MEMORY_CACHE_TTL: NonNegativeInteger
```

Comment:
Cannot be used NonNegativeInteger in here, because the ENV var is a string.
We had
```
Error: Invalid configuration: value ["3600"] at [root.IN_MEMORY_CACHE_TTL] is not a valid [integer >= 0]
```

Reply:
Yep I'll modify it into `NumberFromString`

Reply:
Fixed in [66e5273](https://github.com/pagopa/io-functions-backoffice/pull/17/commits/66e5273edf6e2851642dc6a5eb99400167bb41f0)

Reply:
@AleDore can you add this ENV into the readme pls?